RELEASE_NOTES.md

```diff
@@ -17,115 +17,62 @@ OPAE\'s goal is to accelerate FPGA adoption. It is a community effort to simplif
-While no tools in the OPAE codebase use libopaec++, the code is being kept here
-for any other tools that may use it outside of the OPAE repository.
-This directory, however, will be removed in future versions of OPAE.
-For more information and reference on the official API, see the
-[documentation](https://opae.github.io/latest/docs/fpga_api/fpga_cxx_api.html).
-
--   fpgamux has been ported to use the OPAE Python API.
-
--   In addition to supporting the OPAE driver bundled with OPAE SDK releases, the OPAE SDK libraries now
-support the FPGA driver that has been upstreamed to the Linux Kernel 4.18.
-For more details on this, please see the [OPAE documentation](https://opae.github.io/1.3.0/docs/fpga_dfl_drv/fpga_dfl_drv.html) related to this.
-
--   Seldom in stress tests, kernel panic may be encountered in kernel version 3.10. Preliminary debug information seems to indicate it may be related to hugepage support in the Linux kernel.
-
--   The current Python distributions included in this release are
-    -  opae.fpga-1.3.0.tar.gz - The source files for building the Python bindings. This requires OPAE development package to be installed prior to building
-    -  opae.fpga-1.3.0-cp27-cp27mu-linux_x86_64.whl - A binary package built with Python 2.7
-    -  opae.fpga-1.3.0-cp34-cp34m-linux_x86_64.whl - A binary package built with Python 3.4
-    -  opae.fpga-1.3.0-cp35-cp35m-linux_x86_64.whl - A binary package built with Python 3.5
+- OPAE 1.4.0 may not be compatible with other versions of Linux OS/Kernel
 
+- OPAE & Intel FPGA driver are tested on Intel Programmable Acceleration Card Arria 10 GX FPGA & Intel PAC N3000.
 
--   A different OPN is used in the design examples
+- OPAE & DFL FPGA driver are tested on Intel Programmable Acceleration Card Arria 10 GX FPGA.
 
-    The Intel Quartus Prime Pro Edition license uses a design example
+- FPGA DFL Linux driver source code patchset2 available in Linux 5.4 kernel
+   - https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/drivers?h=linux-5.4.y
 
-    OPN of 10AX115N3F40E2SG, instead of the Intel PAC with Intel Arria
-    10 GX FPGA OPN of 10AX115N2F40E2LG. This difference does not impact
-    your design.
 
--   PCIe directed speed changes are not supported
-
-    Only automatic down-training at boot time is supported
+Notes / Known Issues
+--------------------
 
--   Virtual Function (VF) may fail to attach or detach when using the
-    Linux Red Hat 3.10 kernel
-    
-    This is a known issue with qemu/kvm and libvirt. Refer to the Red
-    Hat website for more information about this issue.
+- In addition to supporting the OPAE driver bundled with OPAE SDK releases, the OPAE SDK libraries now
+support the FPGA driver that has been upstreamed to the Linux Kernel 5.4.
+For more details on this, please see the [OPAE documentation](https://opae.github.io/1.4.0/docs/fpga_dfl_drv/fpga_dfl_drv.html) related to this.
```

Comment:
Getting 404 Not Found for this link.

Reply:
we have run doc gen script to generate 1.4.0 folder.

Reply:
Oh, probably because 1.4.0 doesn't exist yet.  Nevermind.